src/app/core/services/manufacturer.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpParams} from "@angular/common/http";
+import {Observable} from "rxjs";
+import {environment} from "../../../environments/environment";
+import {IManufacturer} from "../models/imanufacturer";
+import {ProductNumberDTO} from "../dto/product-number-dto";
+import {CoreModule} from "../core.module";
+
+@Injectable({
+  providedIn: CoreModule
+})
+export class ManufacturerService {
+
+  constructor(private http: HttpClient) {}
+
+  getManuList(): Observable<Array<IManufacturer>> {
+    return this.http.get<Array<IManufacturer>>(environment.apiUrl + '/manufacturers');
+  }
+
+  countProductsByManus(): Observable<Array<ProductNumberDTO>> {
+    let params = new HttpParams()
+      .set('countOnly', true);
+
+    return this.http.get<Array<ProductNumberDTO>>(environment.apiUrl + '/manufacturers/products', { params: params });
```

Comment:
Sve API URL-ve stavi u neki ts fajl pod nasivom tipa apiURL i stavi unutar Core module-a pod folderom constatns...

I napravi  klase i unutar static readonly stringove gde će ime tog propertija biti neki intuitivni naziv za API endpoint a sama vrednost recimo url adresa.

Primer: static readonly GET_MANUFACTURES_PRODUCTS: string = "/manufacturers/products"

Ovo je zgodno recimo, ako se promeni api endpoint, a koristiš ga na viš mesta u velikim projektima, kao što je to u industriji slučaj... Da ga ne bi menjao svuda, promeniš ga na jednom mestu i to je to...